src/test/java/updated/game/abstractClasses/Gamer.java

```diff
@@ -0,0 +1,43 @@
+package updated.game.abstractClasses;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Getter
+public abstract class Gamer {
+    protected String gamerName;
+    @Setter
+    public char symbol;
+    public int currentScore;
+    public int numberOfMoves;
+
+    public abstract void getSymbolFromUser();
+    public abstract List<Integer> getInputFromUser(Game game);
+    public void reset(){
+        numberOfMoves = 0;
+        symbol = ' ';
+    }
+    public void printDetails(){
+        System.out.println("******************************");
+        System.out.printf("%25s", "Details Received are as: \n");
+        System.out.printf("Username: %s\n", this.gamerName);
+        System.out.printf("Symbol Selected by %s: %c\n", this.gamerName, this.symbol);
+        System.out.println("******************************");
+
+    }
+
```

Comment:
Instead of writing PrintDetails here can can put this method in separate utility class that handles display related logic